app/controllers/admin/export/document_controller.rb

```diff
@@ -55,7 +55,7 @@ def paginated_document_ids
       .joins("INNER JOIN organisations o ON o.id = eo.organisation_id")
       .where(o: { content_id: params.require(:lead_organisation) })
       .where(eo: { lead: true })
-      .where(type: params.require(:type))
+      .by_type_or_subtype(document_type, document_subtype)
```

Comment:
I've made a change so that (as discussed):
- type and invalid subtype = no results
- type, valid subtype and invalid subtype = results for only valid subtype
- type only = results for all subtypes